src/index.html

```diff
@@ -25,6 +25,7 @@
   <title>Digital Excellence</title>
   <base href="/">
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="description" content="Dex provides a platform for students to share and work on projects. Share, work on and create new projects on Dex">
```

Comment:
```suggestion
  <meta name="description" content="DeX provides a platform for students, teachers and employees to share and work on projects. Share, work on and create new projects on DeX">
```

Reply:
```suggestion
  <meta name="description" content="DeX provides a platform for students, teachers and employees to share and work on projects and ideas. Find, create, share and work on projects and ideas on DeX">
```
DeX is about projects AND ideas ;)